packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -20,59 +21,59 @@ const BIGINT1 = BigInt("1");
 export class Len extends Op {
   execute (stack: TEALStack): void {
     this.assertStackLen(stack, 1);
-    const a = this.assertBytes(stack.pop());
-    stack.push(BigInt(a.length));
+    const last = this.assertBytes(stack.pop());
+    stack.push(BigInt(last.length));
   }
 }
 
-// pops two unit64 from stack(a, b) and pushes their sum(a + b) to stack
+// pops two unit64 from stack(last, prev) and pushes their sum(last + prev) to stack
 // panics on overflow (result > max_unit64)
 export class Add extends Op {
   execute (stack: TEALStack): void {
     this.assertStackLen(stack, 2);
-    const a = this.assertBigInt(stack.pop());
-    const b = this.assertBigInt(stack.pop());
-    const result = a + b;
+    const last = this.assertBigInt(stack.pop());
+    const prev = this.assertBigInt(stack.pop());
+    const result = prev + last;
     this.checkOverflow(result);
     stack.push(result);
   }
 }
 
-// pops two unit64 from stack(a, b) and pushes their diff(a - b) to stack
+// pops two unit64 from stack(last, prev) and pushes their diff(last - prev) to stack
 // panics on underflow (result < 0)
 export class Sub extends Op {
   execute (stack: TEALStack): void {
     this.assertStackLen(stack, 2);
```

Comment:
can we change `assertStackLen` to something else ? because this is confusing. this seems as if it is checking if stack length is exactly equal to 2. But that's not the case here.

Reply:
yes, renaming to: `assertMinStackLen`.